src/templates/blog-post.tsx

```diff
@@ -43,10 +43,38 @@ interface IBlogPostPageProps {
 
 const BlogPostPage: React.FC<IBlogPostPageProps> = ({ data }) => {
   const post = data.blogPost
-  const { title, description, picture } = post
+  const {
+    title,
+    description,
+    picture,
+    author: { name },
+    date
+  } = post
   return (
     <>
-      <SEO title={title} description={description} image={picture} />
+      <SEO
+        title={title}
+        description={description}
+        image={picture}
+        meta={[
+          {
+            name: 'twitter:card',
+            content: 'summary_large_image'
```

Comment:
Since our blog posts have some pretty cool art, I updated the twitter card for blogs to have `summary_large_image` instead of `summary`

Current:
![image](https://user-images.githubusercontent.com/43496356/140531038-0734599c-74c1-4c8d-8bfb-f88ccded27dc.png)

Pull Request:

![image](https://user-images.githubusercontent.com/43496356/140530977-aa98d9d9-e733-4d29-8b63-71f40eb8cb8f.png)

While this does look a little cuttoff it does give us large not cutoff previews in Slack and Discord:

![image](https://user-images.githubusercontent.com/43496356/140532853-763bcc95-6d95-400c-955d-f4b39235e9e2.png)

But if we prefer to keep the original, I can take it off 😃